docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -64,6 +67,9 @@ spec:
         automountServiceAccountToken: true
         securityContext:
           runAsUser: 0
+        volumes:
+        - name: agent-data
+          emptyDir: {} # may not be suited for production, the default value is to use host path volume
```

Comment:
I would only do this for Fleet server. Which afaik does not actually need to persist state. While regular Agents running integrations do need this.

Reply:
> I would only do this for Fleet server.

IIUC last commit removed the `emptyDir` on the Fleet server.

Also:
* The use of `emptyDir` is documented in the `Storing local state in host path volume` section, at the end of this document. Should we update it to mention security restrictions, and the fact that it is not required for Fleet Server (which is a TIL for me btw)?
* Should we just not try to use an `hostpath` if Fleet Server is enabled? (to be done in a separate issue/pr ofc)

Reply:
I have not been able to get an answer from the Fleet team on this. However given that we are running Fleet server as a deployment and the chance is high that the on eviction or deletion of the Pod we end up on a different k8s node, I think we are good to merge this as is.